api/src/main/java/com/code4ro/nextdoor/emergency/contact/dto/EmergencyContactDto.java

```diff
@@ -0,0 +1,22 @@
+package com.code4ro.nextdoor.emergency.contact.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
De ce nu folosesti direct @ Data?

Reply:
Am urmarit exemplul care era in proiect, ar arata mai bine cu @Data intr-adevar

Reply:
`@Data` aduce mai multe. Oare avem nevoie de tot ce aduce? eu cred ca nu

Reply:
Nu iti aduce decat hashCode, equals si toString

Reply:
Ceea ce te poate ajuta sa afisez in loguri exact obiectul, poate avem nevoie la un moment dat sa facem equals, mai ales daca vrem sa facem un diff si trebuie sa stim daca sunt egale sau nu. Deci nu vad o problema sa folosim @Data

Reply:
Da, 3 lucruri pe care nu le folosesti. Adica lombok e foarte bun, dar trebuie folosit doar pentru ce ai nevoie, ca sa nu poluezi codul